quotes.yaml

```diff
@@ -23,7 +23,14 @@
 
 - :c: A computer lets you make more mistakes faster than any other invention in human history, with the possible exceptions of handguns and tequila.
   :a: Mitch Ratcliffe
+  
+- :c: But we are HACKERS and hackers have black terminals with green font colors
+  :a: J. Nunemaker   
 
+- :c: Hope That Firewall works because your screwed...
```

Comment:
I'll rectify it all today! Let me know about the white spaces problem, I'm
not getting where exactly white spaces are there! Are you talking about the
extra line? After each quote?
On 18-Jan-2016 12:10 am, "Andri Möll" <notifications@github.com> wrote:

> In quotes.yaml
> <https://github.com/jarmo/hackersays/pull/41#discussion_r49950275>:
>
> >
> > +- :c: Hope That Firewall works because your screwed...
>
> Concur. Not much of an aphorism. >)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jarmo/hackersays/pull/41/files#r49950275>.
>